app/controllers/user/profiles_controller.rb

```diff
@@ -0,0 +1,30 @@
+class User::ProfilesController < ApplicationController
+  before_action :authenticate!
+
+  before_action -> { check_parameters_for :profiles }, only: :update
+
+  def show
+    render_profile
```

Comment:
remove `render_profile` and directly use

```ruby
 render json: :profile
```
Without the `status` parameters, status code will be `200` automatically.